src/Compilers/Test/Core/CompilationVerifier.cs

```diff
@@ -185,7 +185,22 @@ public string DumpIL()
 		/// <param name="expected">The expected IL</param>
         public void VerifyTypeIL(string typeName, string expected)
         {
-            VerifyTypeIL(typeName, output => AssertEx.AssertEqualToleratingWhitespaceDifferences(expected, output, escapeQuotes: false));
+            VerifyTypeIL(typeName, output =>
+            {
+                // All our tests predate ilspy adding `// Header size: ...` to the contents.  So trim that out since we
+                // really don't need to validate superfluous IL comments
+                expected = RemoveHeaderComments(expected);
+                output = RemoveHeaderComments(output);
+
+                AssertEx.AssertEqualToleratingWhitespaceDifferences(expected, output, escapeQuotes: false);
+            });
+        }
+
+        private static readonly Regex s_headerCommentsRegex = new("""^\s*// Header size: [0-9]+\s*$""", RegexOptions.Multiline);
+
+        private static string RemoveHeaderComments(string value)
+        {
+            return s_headerCommentsRegex.Replace(value, "");
```

Comment:
Since we are already doing this, could we also remove the colon from `// Code size: 6 (0x6)` comments to reduce the diffs?

Reply:
that's nto the worst idea :)  But the works already been done.  so i'd prefer to keep as is :)

Reply:
I'm not sure I understand which IL baselines are impacted here. Is the issue that we use ildasm in some places and ICSharpCode.Decompiler in other places? I'm concerned about the inconsistency in baselines and about ongoing PRs being disrupted by the change.

Reply:
Yeah, i'm not sure what a good strategy is here.  As long as the compiler is using this tool, and as long as this tool changes their outpus across releases, then we have this problem.